### ISSUE TITLE
Add README and Project?

### ISSUE THREAD

**Issue by xackery** [fornclake/TetraForce#100]
Howdy, I stumbled upon this project and noticed both of these areas missing. I'm happy to PR a README if you like.

I see there's data in the wiki, but, may not be updated recently.

Let me know what areas you may want some assistance with!

**Comment by josephbmanley** [comment]
Replying for test

**Comment by josephbmanley** [comment]
Hi @xackery! Are you in the discord?

**Comment by xackery** [comment]
Nope. I didn't see the discord link in the README front page ;)

**Comment by josephbmanley** [comment]
True, fair point. Blame @fornclake and @Elginive in a friendly manner please. :+1: 

Discord server: https://discord.gg/P2X2R5T
